docs/Contributing.md

```diff
@@ -76,3 +77,15 @@ it doesn't need to. r=Standard8`
   * master should then be merged with the branch via fast-forward only:
     * `git merge --ff master`
   * If it was successful, push the result
+
+Autolander (bot)
+-------
```

Comment:
Nit: I know this isn't totally necessary, but please could you extend the dashes to the full length of the line above.